package.json

```diff
@@ -9,8 +9,8 @@
     "posttest": "npm run build",
     "build": "sh -c \"rm -rf build/ && NODE_ENV=production babel src/ --out-dir build/\"",
     "lint": "import-sort -l {src,test,public}/**/*.{js,jsx} && eslint --ext js,jsx *.js src/ test/ public/",
-    "lint-fix": "import-sort --write {src,test,public}/**/*.{js,jsx}; eslint --ext js,jsx *.js src/ test/ public/ --fix",
-    "prepublish": "npm run build"
+    "fixlint": "import-sort --write {src,test,public}/**/*.{js,jsx}; eslint --ext js,jsx *.js src/ test/ public/ --fix",
+    "prepare": "npm run build"
```

Comment:
Does `prepare` run automatically before `npm publish`? Because `prepublish` automatically runs whenever you do a `npm publish` so you do not have to do it manually.

Reply:
Yes, [it does](https://docs.npmjs.com/misc/scripts). There was a warning that "prepublish" is deprecated since `npm@4`, which is the reason for this change. It has been replaced by "prepare". See the breaking changes section in the npm 4 release notes: https://github.com/npm/npm/releases/tag/v4.0.0